point-of-sale-app/kubernetes-manifests/mysql-db.yaml

```diff
@@ -0,0 +1,91 @@
+# Copyright 2022 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
```

Comment:
saw that there's a new Yaml file! 
region tag for this:
```
# [START gke_kubernetes_manifests_mysql_db_statefulset_mysql_db]
....
`# [END gke_kubernetes_manifests_mysql_db_statefulset_mysql_db]`
```
around this resource

Reply:
Thank you for catching that. Added the tags in https://github.com/GoogleCloudPlatform/point-of-sale/pull/89/commits/18eb65ab33f82375ecc9c616bb5a9b68d41acddc

Minor change I made to your suggestion: `gke` -> `anthosbaremetal`